website/src/css/custom.css

```diff
@@ -1764,6 +1764,19 @@ section > h2:not(.resource-section) {
 
 
 /* utilities */
+.grid--4-col {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: var(--ifm-spacing-horizontal)
+}
+
+@media(min-width: 996px) {
+  .grid--4-col {
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: calc(var(--ifm-spacing-horizontal) * 2);
+  }
+}
```

Comment:
Testing locally, it looks like the grid of 4 runs out of room around `1500px` wide. So in between ~996px and ~1500px, the cards will start to be pushed out of the container and off the screen.

Adding another media query below the 996px media query could solve this. Rather than having 4 columns down to a min-width of 996px, I recommend creating another media query below this one with a min-width of 1500px, and have the `grid-template-columns: repeat(4, 1fr);` css property set to 4 within this 1500px media query.

Then for this 996px media query, I would change the `grid-template-columns` property from 4 to 2. So on screens 1500px or wider, it will have 4 columns, then from 996px to 1499px it will have 2 columns, then 1 column for anything below 996px.

Let me know if you have any questions!

![image](https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/36694aef-06ae-4621-b87e-218d378edb2f)

Reply:
great callout, thank you so much @JKarlavige ! ok I've amended that on my end and i also added a 5-column grid option as well, however i don't think we'll use that one much - is that ok?

also also - i tried synching changes and it brought over a ton of changes so apologies for all the increased file changes it brought over! not sure why it did that!